config/default-manifest.json

```diff
@@ -214,6 +211,7 @@
         "uncommitted": null,
         "storage_domain_type": null,
         "host_storages": {
+          "ems_ref": null,
```

Comment:
@pemcg this was intentionally removed from master because host_storages no longer have this column